FrontEnd/src/components/ValidationTable.tsx

```diff
@@ -29,28 +38,33 @@ export default function ValidationTable({
           <table>
             <thead>
               <tr>
-                {headers.map((row) => {
-                  // display headers in table row using html array map
-                  return <td>{row.label}</td>;
+                {headers.map((row, index) => {
+                  return <td key={'header' + index}>{row.label}</td>;
                 })}
               </tr>
             </thead>
           </table>
 
           <div>
-            {data.map((data, index) => {
+            {uploadedData.map((uploadedData: dataStructure, index) => {
```

Comment:
Not a mistake but I would choose a different name for the element of uploadedData arrayinside map function, for example {uploadedData.map((anyOtherName: dataStructure, index), for better readability.

Reply:
Just fixed it.